IFComp/lib/IFComp/Controller/Admin/Feedback.pm

```diff
@@ -0,0 +1,34 @@
+package IFComp::Controller::Admin::Feedback;
+use Moose;
+use namespace::autoclean;
+
+BEGIN { extends 'Catalyst::Controller'; }
+
+sub index : Chained('/admin/root') : PathPart( 'feedback') : Args(0) {
+    my ( $self, $c ) = @_;
+
+    unless ( $c->user && $c->check_any_user_role('curator') ) {
+        $c->log->debug("get outta here");
```

Comment:
Please get rid of this line, or improve the log message and add `if ($c->debug)` to the line.